synapse/models/risk.py

```diff
@@ -144,6 +147,25 @@ def getModelDefs(self):
                     }),
                 )),
 
+                ('risk:alert', {}, (
```

Comment:
How will the alert be linked back to an IP (for network alerts) or a host (for host-based alerts)?

Reply:
light edges for now